### PR TITLE
Add/Update German translation

### PR DESCRIPTION
**Pull request by gonzoMD**
## Purpose

Add some new translations and improve a few existing ones.

## Proposed changes

**Add translations for:**
- [CHARMAP_NEW] 
- [COMP] 
- [EVENTCREATE] 
- [TIMEOUT] 
- [WHERE] 
- [UTILMAN]
- [CHKDSK] 
- [SHELLEXT]: cryptext, devcpux, ~~fontext~~, mycomp, netplwiz
- [MSXML3R]
- [THEMES]: Blackshade, Lunar

**Update translations for:**
- [CLIPBRD] 
- [SHORTCUTS.INF] 

## TODO

Some other German native speakers should proof read this.

### CODE REVIEW

File: base/applications/cmdutils/where/lang/de-DE.rc [hunk 1]
```diff
@@ -0,0 +1,53 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (binarymaster) [inline base/applications/cmdutils/where/lang/de-DE.rc:1]:
```suggestion
LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL
```

File: dll/shellext/devcpux/lang/de-DE.rc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    LTEXT "Modell:", -1, 7, 49, 50, 8
+    LTEXT "<test>", IDC_MODEL, 7, 60, 238, 8
+
+    LTEXT "Stepping:", -1, 7, 75, 50, 8
+    LTEXT "<test>", IDC_STEPPING, 7, 86, 238, 8
+
+    LTEXT "Core Geschwindigkeit:", -1, 7, 101, 50, 8
+    LTEXT "<test>", IDC_CORESPEED, 7, 112, 238, 8
+END
+*/
```
Comment (binarymaster) [inline dll/shellext/devcpux/lang/de-DE.rc:41]: No newline at end of file

File: dll/shellext/fontext/lang/de-DE.rc [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Don't translate this file at current time
```
Comment (binarymaster) [inline dll/shellext/fontext/lang/de-DE.rc:1]: :thinking:

File: media/themes/Blackshade/blackshade.msstyles/lang/de-DE.rc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+{
+5000, "Black Shade"
+5001, "Visueller Stil"
+5002, "ReactOS Foundation"
+5003, "Grafiken erstellt durch David Quintana <gigaherz@gmail.com>"
+5004, "Grafiken veröffentlicht unter CC-BY-SA 4.0"
+5005, "https://reactos.org/"
+5006, "1.0"
+5007, "Visueller Stil für ReactOS"
+}
```
Comment (binarymaster) [inline media/themes/Blackshade/blackshade.msstyles/lang/de-DE.rc:37]: No newline at end of file

File: dll/shellext/fontext/lang/de-DE.rc (hunk 3 above)
Comment (gonzoMD) [reply dll/shellext/fontext/lang/de-DE.rc:1]: It's not the only translation where is information was ignored

File: base/applications/cmdutils/where/lang/de-DE.rc (hunk 1 above)
Comment (gonzoMD) [reply base/applications/cmdutils/where/lang/de-DE.rc:1]: right. *facepalm* fixed.

File: dll/shellext/devcpux/lang/de-DE.rc (hunk 2 above)
Comment (gonzoMD) [reply dll/shellext/devcpux/lang/de-DE.rc:41]: is that important? even the English one (which I copied) did not have a newline.

File: dll/shellext/fontext/lang/de-DE.rc (hunk 3 above)
Comment (binarymaster) [reply dll/shellext/fontext/lang/de-DE.rc:1]: Ah if it's the same, then it's okay.

File: base/applications/cmdutils/timeout/lang/de-DE.rc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+\n\
+    /NOBREAK  Ignoriert sämtliche Tasteneingaben mit Ausnahme von Strg+C.\n\
+"
+    IDS_ERROR_OUT_OF_RANGE "FEHLER: Die Zeit muss innerhalb des Wertebereiches liegen (-1 to 99999).\n"
+    IDS_ERROR_INVALID_HANDLE_VALUE "FEHLER: Standard-Handle der Konsole nicht verfügbar (error %lu).\n"
+    IDS_ERROR_READ_INPUT "FEHLER: Abfragen der Konsoleneingabe nicht möglich (error %lu).\n"
+    IDS_ERROR_NO_TIMER_VALUE "FEHLER: Es muss eine Zeit angegeben werden (-1 to 99999).\n"
+    IDS_ERROR_ONE_TIME "FEHLER: Es wird nur ein Zeitwert benötigt.\n"
+    IDS_NOBREAK_INPUT "Drücken Sie Strg+C zum Beenden..."
+    IDS_USER_INPUT "Taste drücken um fortzufahren..."
```
Comment (Gamemap) [inline base/applications/cmdutils/timeout/lang/de-DE.rc:30]:
```suggestion
    IDS_USER_INPUT "Beliebige Taste drücken um fortzufahren..."
```
If any keypress is accepted I think this formulation is clearer.

File: base/applications/cmdutils/timeout/lang/de-DE.rc [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    IDS_NOBREAK_INPUT_COUNT "Warte %d Sekunde(n), Drücken Sie Strg+C zum Beenden..."
+    IDS_USER_INPUT_COUNT "Warte %d Sekunde(n), Taste Drücken um fortzufahren..."
```
Comment (Gamemap) [inline base/applications/cmdutils/timeout/lang/de-DE.rc:32]:
```suggestion
    IDS_USER_INPUT_COUNT "Warte %d Sekunde(n), Beliebige Taste Drücken um fortzufahren..."
```
If any keypress is accepted I think this formulation is clearer.

**Review by Gamemap — CHANGES_REQUESTED** [review]
Since this is my first rewiew please excuse me if I make a mistake.
(I'm a native German speaker.)

File: dll/shellext/fontext/lang/de-DE.rc (hunk 3 above)
Comment (learn-more) [reply dll/shellext/fontext/lang/de-DE.rc:1]: So let's make it worse by also ignoring it.
Guess I'll either never update this shellext again, or ignore all languages.

File: dll/shellext/devcpux/lang/de-DE.rc (hunk 2 above)
Comment (learn-more) [reply dll/shellext/devcpux/lang/de-DE.rc:41]: it's good practice to add it.

File: base/applications/cmdutils/eventcreate/lang/de-DE.rc [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+                          erstellt wird. Zulässige Namen sind:\n\
+                          Application, System, Security\n\
+                          (Letzteres ist dem SYSTEM-Account vorbehalten).\n\
+\n\
+    /SO Quelle            Bestimmt den Namen der Quelle des Ereignisses\n\
+                          (Wenn nicht angegeben, wird der Standardname\n\
+                          'eventcreate' benutzt).\n\
+                          Eine gültige Quelle kann aus einer beliebigen\n\
+                          Zeichenfolge bestehen und sollte die Anwendung\n\
+						  oder Komponente darstellen, die das Ereignis verursacht.\n\
```
Comment (dreimer1986) [inline base/applications/cmdutils/eventcreate/lang/de-DE.rc:34]: Is that one missing two spaces?

File: base/applications/charmap_new/lang/de-DE.rc [hunk 8]
```diff
@@ -0,0 +1,22 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (dreimer1986) [inline base/applications/charmap_new/lang/de-DE.rc:1]: LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL

Comment (gonzoMD) [reply base/applications/charmap_new/lang/de-DE.rc:1]: Yeah. I already fixed that but g(sh)it reverted it. Sigh.

File: base/applications/cmdutils/where/lang/de-DE.rc [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    IDS_WANT_VALUE "FEHLER: Ein Wert wird für '%ls' benötigt.\n"
+    IDS_TYPE_HELP "Tippen Sie ""WHERE /?"" für eine Benutzungshilfe.\n"
+    IDS_ENVPAT_WITH_R "FEHLER: ""$env:pattern""kann nicht mit /R benutzt werden.\n"
+    IDS_PATHPAT_WITH_R "FEHLER: ""path:pattern"" Format kann nicht mit /R benutzt werden.\n"
+    IDS_BAD_PATHPAT "FEHLER: Ungültiges Suchmuster ist in ""path:pattern"" angegeben.\n"
+    IDS_OUTOFMEMORY "FEHLER: Ungenügender Speicher.\n"
+    IDS_BAD_ENVVAR "FEHLER: Umgebungsvariable ""%ls"" wurde nicht gefunden.\n"
+    IDS_CANT_FOUND "FEHLER: Das System konnte die angegebene Datei nicht finden.\n"
+    IDS_BAD_DIR "FEHLER: Ungültiges Verzeichnis angegeben.\n"
+    IDS_BAD_NAME "FEHLER: Die Syntax der Datei-/Verzeichnisname(n) oder Datenträgerbezichnung ist falsch.\n"
```
Comment (JoachimHenze) [inline base/applications/cmdutils/where/lang/de-DE.rc:51]: bezichnung->bezeichnung

File: base/applications/utilman/umandlg/lang/de-DE.rc [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_CONTEXTHELP
+CAPTION "Eingabehilfen-Manager"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LISTBOX IDC_LISTBOX, 4, 4, 273, 56, LBS_STANDARD | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_BORDER
+    CONTROL "", IDC_GROUPBOX, "Button", BS_GROUPBOX | WS_CHILD | WS_VISIBLE, 3, 62, 275, 92
+    CONTROL "Start", IDC_START, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 14, 76, 45, 16
+    CONTROL "Stop", IDC_STOP, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 69, 76, 45, 16
+    CONTROL "Starte automatisch, wenn ich mich anmelde", IDC_START_LOG_IN, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 101, 206, 14
```
Comment (JoachimHenze) [inline base/applications/utilman/umandlg/lang/de-DE.rc:19]: Starte automatisch bei der Anmeldung

File: base/applications/utilman/umandlg/lang/de-DE.rc [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+EXSTYLE WS_EX_CONTEXTHELP
+CAPTION "Eingabehilfen-Manager"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LISTBOX IDC_LISTBOX, 4, 4, 273, 56, LBS_STANDARD | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_BORDER
+    CONTROL "", IDC_GROUPBOX, "Button", BS_GROUPBOX | WS_CHILD | WS_VISIBLE, 3, 62, 275, 92
+    CONTROL "Start", IDC_START, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 14, 76, 45, 16
+    CONTROL "Stop", IDC_STOP, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 69, 76, 45, 16
+    CONTROL "Starte automatisch, wenn ich mich anmelde", IDC_START_LOG_IN, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 101, 206, 14 
+    CONTROL "Starte automatisch wenn ich meinen Desktop sperre", IDC_START_DESKTOP, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 118, 212, 14
```
Comment (JoachimHenze) [inline base/applications/utilman/umandlg/lang/de-DE.rc:20]: Starte automatisch bei Sperrung des Desktops

File: base/applications/utilman/umandlg/lang/de-DE.rc [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+CAPTION "Eingabehilfen-Manager"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LISTBOX IDC_LISTBOX, 4, 4, 273, 56, LBS_STANDARD | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_BORDER
+    CONTROL "", IDC_GROUPBOX, "Button", BS_GROUPBOX | WS_CHILD | WS_VISIBLE, 3, 62, 275, 92
+    CONTROL "Start", IDC_START, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 14, 76, 45, 16
+    CONTROL "Stop", IDC_STOP, "Button", BS_PUSHBUTTON | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 69, 76, 45, 16
+    CONTROL "Starte automatisch, wenn ich mich anmelde", IDC_START_LOG_IN, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 101, 206, 14 
+    CONTROL "Starte automatisch wenn ich meinen Desktop sperre", IDC_START_DESKTOP, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 118, 212, 14 
+    CONTROL "Starte automatisch wenn der Eingabehilfen-Manager startet", IDC_START_UTILMAN, "Button", BS_CHECKBOX | WS_CHILD | WS_VISIBLE | WS_DISABLED | WS_TABSTOP, 12, 134, 212, 13
```
Comment (JoachimHenze) [inline base/applications/utilman/umandlg/lang/de-DE.rc:21]: Starte automatisch beim Start des Eingabehilfen-Managers

File: base/system/chkdsk/lang/de-DE.rc [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+    IDS_FILE_SYSTEM "Der Dateisystemtyp ist %s.\n"
+    IDS_ABOUT "\n\
+Chkdskx v1.0.1 by Mark Russinovich\n\
+Systems Internals - http://www.sysinternals.com\n\
+ReactOS adaptation 1999 by Emanuele Aliberti\n\n"
+END
+
+STRINGTABLE
+BEGIN
+    IDS_VOLUME_IN_USE "Das Laufwerk ist in Benutzung und kann nicht gesperrt werden\n"
```
Comment (JoachimHenze) [inline base/system/chkdsk/lang/de-DE.rc:26]: either end each of those with a point or none

File: base/system/chkdsk/lang/de-DE.rc [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    IDS_CHKDSK_FAIL "Chkdsk konnte den Vorgang nicht erfolgreich ausführen.\n\n"
+    IDS_NO_ENTRY_POINT "FMIFS-Einstiegspunkte konnten nicht gefunden werden .\n\n"
```
Comment (JoachimHenze) [inline base/system/chkdsk/lang/de-DE.rc:28]: Omit the space character before the point

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: base/applications/cmdutils/eventcreate/lang/de-DE.rc (hunk 7 above)
Comment (gonzoMD) [reply base/applications/cmdutils/eventcreate/lang/de-DE.rc:34]: Tabs vs. Spaces. Fixed.

File: base/applications/cmdutils/eventcreate/lang/de-DE.rc [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+    IDS_BADSYNTAX_4 "Fehlerhafte Kommandozeilensyntax. Der Wert kann der Option '%s' nicht zugewiesen werden.\n"
+    IDS_BADSYNTAX_5 "Fehlerhafte Kommandozeilensyntax. Die Option '%s' ist nicht mehr als %lu mal erlaubt.\n"
+    IDS_BADSYNTAX_6 "Fehlerhafte Kommandozeilensyntax. Die zwingende Option '%s' fehlt.\n"
+    // IDS_BADSYNTAX_7 "Fehlerhafte Kommandozeilensyntax. Der Wert für die Option '%s' ist außerhalb des Wertebereiches.\n"
+    IDS_BADSYNTAX_7 "Fehlerhafte Kommandozeilensyntax. Der Wert für die Option '%s' muss zwischen %d und %d. liegen\n"
+
+    IDS_LOG_NOT_FOUND "Das Protokoll '%s' existiert nicht. Das Ereignis konnte nicht erstellt werden.\n"
+    IDS_SOURCE_NOCREATE "Die neue Quelle kann nicht erstellt werden, da kein Protokollname angegebenen wurde.\nNutzen Sie die Option /L um einen Protokollnamen anzugeben.\n"
+    IDS_SOURCE_EXISTS "Die Quelle existiert bereits im Protokoll '%s' und kann nicht dupliziert werden.\n"
+    IDS_SOURCE_NOT_CUSTOM "Der Quellparameter wird benutzt um benutzerdefinierte Scripte/Anwendungen zu identifizieren\n(keine installierten Anwendungen).\n"
```
Comment (tkreuzer) [inline base/applications/cmdutils/eventcreate/lang/de-DE.rc:63]: Deutsch wäre ja eigentlich "Skripte".

File: base/applications/cmdutils/timeout/lang/de-DE.rc [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+    /?        Zeigt diesen Hilfebildschirm an.\n\
+\n\
+    /T Zeit   Bestimmt die zu wartenden Sekunden (zwischen -1 und 99999).\n\
+              Ein Wert von -1 bedeutet, dass das Programm auf einen Tastendruck wartet.\n\
+              Beachten Sie, dass die Angabe von ""/T"" optional ist, Sie können die Zeit\n\
+              ohne dessen Angabe übergeben.\n\
+\n\
+    /NOBREAK  Ignoriert sämtliche Tasteneingaben mit Ausnahme von Strg+C.\n\
+"
+    IDS_ERROR_OUT_OF_RANGE "FEHLER: Die Zeit muss innerhalb des Wertebereiches liegen (-1 to 99999).\n"
```
Comment (tkreuzer) [inline base/applications/cmdutils/timeout/lang/de-DE.rc:24]: `(-1 bis 99999)`

File: base/applications/cmdutils/timeout/lang/de-DE.rc [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+    IDS_ERROR_INVALID_HANDLE_VALUE "FEHLER: Standard-Handle der Konsole nicht verfügbar (error %lu).\n"
```
Comment (tkreuzer) [inline base/applications/cmdutils/timeout/lang/de-DE.rc:25]: `Fehlercode %lu`?

File: base/applications/cmdutils/where/lang/de-DE.rc [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+  /R dir   Startet die Suche vom angegebenen Verzeichnis\n\
+           und durchsucht dieses rekursiv.\n\
+  /T       Zeigt die Dateigröße und den letzten Änderungszeitpunkt\n\
+           aller übereinstimmenden Dateien an.\n\
+  Muster   Bestimmt das Suchmuster für die gesuchten Dateien.\n\
+           Platzhalter * und ? können im Muster verwendet werden.\n\
+           Formate ""$env:pattern""  und ""path:pattern"" können\n\
+           auch angegeben werden, wenn ""env"" eine Umgebungsvariable ist,\n\
+           und die Suche in den angegebenen Pfaden der Umgebungsvariable\n\
+           ""env"" ausgeführt wird. Diese Formate sollten nicht mit\n\
```
Comment (tkreuzer) [inline base/applications/cmdutils/where/lang/de-DE.rc:24]: `können nicht mit /R verwendet werden`? Das geht zumindestens aus einer Fehlermeldung weiter unten hervor.

File: dll/shellext/devcpux/lang/de-DE.rc [hunk 19]
```diff
@@ -0,0 +1,41 @@
+LANGUAGE LANG_GERMAN, SUBLANG_NEUTRAL
+
+DLG_PROCESSORINFO DIALOGEX 0, 0, 252, 218
+STYLE DS_SHELLFONT | WS_POPUP | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME
+CAPTION "CPU"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "Merkmale:", -1, 7, 23, 50, 8
+    LTEXT "test", IDC_FEATURES, 57, 23, 181, 8
```
Comment (tkreuzer) [inline dll/shellext/devcpux/lang/de-DE.rc:9]: "test"?

File: dll/shellext/devcpux/lang/de-DE.rc [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+END
+
+/*
+DLG_PROCESSORINFO DIALOGEX 0, 0, 252, 218
+STYLE DS_SHELLFONT | WS_POPUP | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_THICKFRAME
+CAPTION "CPU"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "Merkmale:", -1, 7, 23, 50, 8
+    LTEXT "<test>", IDC_FEATURES, 7, 34, 238, 8
```
Comment (tkreuzer) [inline dll/shellext/devcpux/lang/de-DE.rc:30]: "<test>"? Ist igendwie inkonsistent mit dem oben.

**Review by tkreuzer — APPROVED** [review]
Passt schon.

**Review by JoachimHenze — APPROVED** [review]

**Review by Gamemap — APPROVED** [review]

File: dll/shellext/devcpux/lang/de-DE.rc (hunk 19 above)
Comment (gonzoMD) [reply dll/shellext/devcpux/lang/de-DE.rc:9]: Ja. ist in den anderen Übersetzungen auch so, daher hab ich es so gelassen.

**Pull request merged**